src/transformers/modeling_utils.py

```diff
@@ -5083,6 +5083,12 @@ def loss_function(self):
             loss_type = "ForCausalLM"
         return LOSS_MAPPING[loss_type]
 
+    @property
+    def compiled_forward(self):
+        if not hasattr(self, "_compiled_forward"):
+            self._compiled_forward = torch.compile(self.__call__, mode="reduce-overhead", fullgraph=True)
+        return self._compiled_forward
```

Comment:
Cool, let's maybe document this a bit? (*usage)
```suggestion
    @property
    def compiled_call(self):
        if not hasattr(self, "_compiled_call"):
            self._compiled_call = torch.compile(self.__call__, mode="reduce-overhead", fullgraph=True)
        return self._compiled_call
```

Reply:
Haha yes, I was asking myself whether I wanted to name it `compiled_forward` or `compiled_call`... Updated accordingly!